src/usr/local/containerbase/tools/v2/rye.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+function install_tool () {
+  local versioned_tool_path
+  local arch
+  local name
+  local version
+  local repo
+
+  arch=${ARCHITECTURE}
+  name=$TOOL_NAME
+  version=$TOOL_VERSION
+  repo="astral-sh/$name"
+
+  binary="${name}-${arch}-linux"
+  file_name="${binary}.gz"
+  base_url="https://github.com/${repo}/releases/download/${version}"
+  file_url="${base_url}/${file_name}"
+
+  checksum_file=$(get_from_url "${file_url}.sha256")
+  # get checksum from file
+  expected_checksum=$(cut -d' ' -f1  "${checksum_file}")
+
+  file=$(get_from_url "${file_url}" "${file_name}" "${expected_checksum}" "sha256sum")
+
+  versioned_tool_path=$(create_versioned_tool_path)
+  create_folder "${versioned_tool_path}/bin"
+
+  bsdtar -x -f "${file}" -C "${TEMP_DIR}/rye"
```

Comment:
ci is failing because of corrupt / unreadable archive

Reply:
That's because bsdtar doesn't actually support plain .gz archives